client/messageprocessor.js

```diff
@@ -73,7 +73,7 @@ function process(in_msg, callbacktopic) {
     var envid = "";
     
 
-    if (msg.exitcode && msg.exitcode != 0) {
+    if (msg.exitcode && msg.exitcode != 0 && !msg.silentlyContinue) {
```

Comment:
Sure, it was as described in the PR description. I will put it in the docs in my next iteration of checkin